### PR TITLE
updated dependencies

### PR DESCRIPTION
**Pull request by scottinet**
* Updated dependencies
* Updates cucumber features to their new hook format

### CODE REVIEW

**Comment by j33f** [comment]
+1

**Comment by benoitvidis** [comment]
+1

File: package.json [hunk 1]
```diff
@@ -32,49 +32,49 @@
...
+    "lodash": "4.11.1",
     "ms": "^0.7.1",
-    "newrelic": "1.26.0",
+    "newrelic": "1.26.2",
     "ngeohash": "0.6.0",
     "node-units": "0.1.7",
     "node-uuid": "1.4.7",
     "passport": "0.3.2",
-    "pm2": "1.1.1",
+    "pm2": "1.1.2",
```
Comment (dbengsch) [inline package.json]: The docker image might need to be re-built to be in sync with this update

**Comment by stafyniaksacha** [comment]
+1 (need to resolve conflicts before merge)

**Pull request merged**